py/desisim/scripts/quickquasars.py

```diff
@@ -102,7 +104,7 @@ def parse(options=None):
 
     parser.add_argument('--mags', action = "store_true", help="DEPRECATED; use --bbflux")
 
-    parser.add_argument('--bbflux', action = "store_true", help="compute and write the QSO broad-band fluxes in the fibermap")
+    parser.add_argument('--bbflux', type=bool, default=True, help="compute and write the QSO broad-band fluxes in the fibermap ")
```

Comment:
Ups! I think I accidentally pushed an unrelated change... should I included in this PR or left it aside?

Reply:
I'm ok putting it all together, it is a small fix.